docs/ref/forms/api.txt

```diff
@@ -830,107 +830,203 @@ The field-specific output honors the form object's ``auto_id`` setting::
     >>> print(f['message'])
     <input type="text" name="message" id="id_message" />
 
-For a field's list of errors, access the field's ``errors`` attribute.
+Attributes of ``BoundField``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. attribute:: BoundField.errors
+.. attribute:: BoundField.auto_id
 
-A list-like object that is displayed as an HTML ``<ul class="errorlist">``
-when printed::
+    Calculates and returns the ID attribute for this ``BoundField`` if the
+    associated form has an :attr:`~django.forms.Form.auto_id`. Returns an
+    empty string otherwise:
 
-    >>> data = {'subject': 'hi', 'message': '', 'sender': '', 'cc_myself': ''}
-    >>> f = ContactForm(data, auto_id=False)
-    >>> print(f['message'])
-    <input type="text" name="message" />
-    >>> f['message'].errors
-    ['This field is required.']
-    >>> print(f['message'].errors)
-    <ul class="errorlist"><li>This field is required.</li></ul>
-    >>> f['subject'].errors
-    []
-    >>> print(f['subject'].errors)
+    .. code-block:: html+django
 
-    >>> str(f['subject'].errors)
-    ''
+        <!-- wrap this field in a div with an unique ID -->
+        <div id="{{ field.auto_id }}">
+            <!-- don't actually render the field with a widget -->
+            <input type="hidden" name="{{ field.html_name }}" value="hidden value" />
+        </div>
 
-.. method:: BoundField.label_tag(contents=None, attrs=None, label_suffix=None)
+    .. note::
+       This attribute is calculated by the form only, so if the field's widget
```

Comment:
This part is still confusing. Can we provide a different attribute that's always correct? Should it return something like `widget.attrs.get('id') or self.auto_id`?